.github/workflows/main.yml

```diff
@@ -203,7 +203,8 @@ jobs:
         working-directory: libs/sdk-react-native 
         run: |
           yarn global add tslint typescript
-          brew install kotlin ktlint swiftformat
+          brew install kotlin swiftformat
+          curl -sSLO https://github.com/pinterest/ktlint/releases/download/1.1.1/ktlint && chmod a+x ktlint && sudo mv ktlint /usr/local/bin/
```

Comment:
Perhaps we can use homebrew to install the specific version?

`brew install ktlint@1.1.1 `

Reply:
I tried that in #759 but it didn't work.

Reply:
Looks like that only works if the homebrew formula has multiple versions published. For example: https://formulae.brew.sh/formula/postgresql@12

For `ktlint` the formula has a single version: https://formulae.brew.sh/formula/ktlint , so that syntax won't work for it.

Reply:
I missed that.